### PR TITLE
[docs] Update docs about the direction metrics attribute transition

### PR DESCRIPTION
**Pull request by dmitryax**
Hold off the transition and update the documentation according to https://github.com/open-telemetry/opentelemetry-specification/issues/2726

Replaces https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13256

### CODE REVIEW

File: receiver/elasticsearchreceiver/README.md [hunk 1]
```diff
@@ -65,28 +64,8 @@ The following feature gates control the transition process:
...
-- `receiver.elasticsearchreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
-- `receiver.elasticsearchreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
-
-3. Phase 3, version and date TBD:
-
-- The feature gates are removed.
-- The new metrics without `direction` attribute are always emitted.
-- The deprecated metrics with `direction` attribute are no longer available.
-
+The final decision on the transition is not finalized yet. The transition is on hold until
```
Comment (TylerHelmuth) [inline receiver/elasticsearchreceiver/README.md:67]: I would leave the "Transition Schedule" header so existing links to the doc still work.

File: receiver/hostmetricsreceiver/README.md [hunk 2]
```diff
@@ -182,33 +181,8 @@ The following feature gates control the transition process:
...
-- The new metrics are enabled by default, deprecated metrics disabled, they can be enabled with the feature gates.
-- `receiver.hostmetricsreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
-- `receiver.hostmetricsreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
-
-4. v0.60.0, September 2022:
-
-- The feature gates are removed.
-- The new metrics without `direction` attribute are always emitted.
-- The deprecated metrics with `direction` attribute are no longer available.
+The final decision on the transition is not finalized yet. The transition is on hold until
```
Comment (TylerHelmuth) [inline receiver/hostmetricsreceiver/README.md:184]: Same here

File: receiver/memcachedreceiver/README.md [hunk 3]
```diff
@@ -66,26 +65,8 @@ The following feature gates control the transition process:
...
-- The new metrics are enabled by default, deprecated metrics disabled, they can be enabled with the feature gates.
-- `receiver.memcachedreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
-- `receiver.memcachedreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
-
-3. v0.60.0, September 2022:
-
-- The feature gates are removed.
-- The new metrics without `direction` attribute are always emitted.
-- The deprecated metrics with `direction` attribute are no longer available.
+The final decision on the transition is not finalized yet. The transition is on hold until
```
Comment (TylerHelmuth) [inline receiver/memcachedreceiver/README.md:68]: And here

File: receiver/vcenterreceiver/README.md [hunk 4]
```diff
@@ -88,26 +87,8 @@ The following feature gates control the transition process:
...
-- The new metrics are enabled by default, deprecated metrics disabled, they can be enabled with the feature gates.
-- `receiver.vcenterreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
-- `receiver.vcenterreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
-
-3. v0.60.0, September 2022:
-
-- The feature gates are removed.
-- The new metrics without `direction` attribute are always emitted.
-- The deprecated metrics with `direction` attribute are no longer available.
+The final decision on the transition is not finalized yet. The transition is on hold until
```
Comment (TylerHelmuth) [inline receiver/vcenterreceiver/README.md:90]: And here

File: receiver/zookeeperreceiver/README.md [hunk 5]
```diff
@@ -45,26 +43,8 @@ The following feature gates control the transition process:
...
-- The new metrics are enabled by default, deprecated metrics disabled, they can be enabled with the feature gates.
-- `receiver.zookeeperreceiver.emitMetricsWithDirectionAttribute` is disabled by default.
-- `receiver.zookeeperreceiver.emitMetricsWithoutDirectionAttribute` is enabled by default.
-
-3. v0.60.0, September 2022:
-
-- The feature gates are removed.
-- The new metrics without `direction` attribute are always emitted.
-- The deprecated metrics with `direction` attribute are no longer available.
+The final decision on the transition is not finalized yet. The transition is on hold until
```
Comment (TylerHelmuth) [inline receiver/zookeeperreceiver/README.md:46]: And here

File: receiver/elasticsearchreceiver/README.md (hunk 1 above)
Comment (dmitryax) [reply receiver/elasticsearchreceiver/README.md:69]: Added

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**